src/Evolution/DiscontinuousGalerkin/InitializeElement.hpp

```diff
@@ -90,6 +90,16 @@ struct InitializeElement {
   template <typename Tag>
   using boundary_tag = Tags::Interface<Tags::BoundaryDirections<Dim>, Tag>;
 
+  // It would be good to decouple this from the Metavariables entirely. This
```

Comment:
Why would this be good? You say what you think is necessary to do it but I have no idea why you want to do it.